### PR TITLE
Compatibility with swift playgrounds

### PR DESCRIPTION
**Pull request by macteo**
- Added missing `import CoreGraphics` where needed.
- Added missing `import UIKit` where needed.

### CODE REVIEW

**Comment by petester42** [comment]
Neat idea. Maybe you could add a playground the repo and we could use that instead of the demo project. Good work 👌

**Comment by liuxuan30** [comment]
@petester42 we cool to merge some PRs like this? I think @danielgindi is too busy to review all the PRs :)

BTW, Seeing Travis new error type:
>xcrun instruments -w '22FA2149-1241-469C-BF6D-462D3837DB72' || sleep 15
Instruments Usage Error: Unknown device specified: "22FA2149-1241-469C-BF6D-462D3837DB72"

Is this travis issue?

**Comment by macteo** [comment]
I tried to solve the travis configuration error, unfortunately without success. Many PR are failing due to the same issue.

**Comment by liuxuan30** [comment]
@macteo did you just try to replace the device id with any one in the list, for example,
>iPhone 7 (10.2) [34FA4749-C467-4D45-9F8E-E31AEDDC39C2] (Simulator)

**Comment by petester42** [comment]
Ya that error is probably because some simulators got removed with Xcode 8.3. I'll look at this pull request soon and merge.

**Comment by liuxuan30** [comment]
@petester42 I update the device id in #2378

File: Source/Charts/Data/Interfaces/IChartDataSet.swift [hunk 1]
```diff
@@ -10,6 +10,7 @@
 //
 
 import Foundation
+import CoreGraphics
```
Comment (liuxuan30) [inline Source/Charts/Data/Interfaces/IChartDataSet.swift]: It's interesting in my Xcode, it never warns or complains missing CoreGraphics, though IChartDataSet refer CGPoint class

Comment (liuxuan30) [inline Source/Charts/Data/Interfaces/IChartDataSet.swift]: It's interesting in my Xcode, it never warns or complains missing CoreGraphics, though IChartDataSet refer CGPoint class

Comment (petester42) [inline Source/Charts/Data/Interfaces/IChartDataSet.swift]: Since UIKit and AppKit imports CoreGraphics it is not needed.

Comment (macteo) [inline Source/Charts/Data/Interfaces/IChartDataSet.swift]: Until you try to use in a Playground

Comment (liuxuan30) [inline Source/Charts/Data/Interfaces/IChartDataSet.swift]: @macteo shall we consider it's playground bug?

Comment (macteo) [inline Source/Charts/Data/Interfaces/IChartDataSet.swift]: I don't think so as it seems a strict requirement.

**Pull request merged**